plugin.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <plugin xmlns="http://apache.org/cordova/ns/plugins/1.0"
         id="cordova-plugin-indooratlas"
-        version="1.7.0">
+        version="1.7.1">
```

Comment:
Should be 1.8.0 since this adds new functionality is added (see https://semver.org/)